### ISSUE TITLE
[Rule] Java: cookie missing same-site false positive

### ISSUE THREAD

**Issue by peter17** [returntocorp/sgrep-rules#2533]
**Describe the bug**
The rule java/lang/security/audit/cookie-missing-samesite.yaml returns a false positive when the cookie is set to null.

**To Reproduce**

```
	public boolean unsetCookie(final HttpServletResponse response) throws Exception {
		response.setHeader("Set-Cookie", null);
		return true;
	}
```
This triggers the rule but should not because there is no point in using the SameSite attribute when unsetting the cookie.

**Expected behavior**
As the cookie is set to "null" (i.e. unset), it should not be reported as missing the SameSite attribute

**Priority**
How important is this to you?
- [ ] P0: blocking me from making progress
- [ ] P1: this will block me in the near future
- [X] P2: annoying but not blocking me

**Additional Context**
There may be additional values to exclude, such as setting the cookie header to an empty string.

**Comment by 0xDC0DE** [comment]
Thank you @peter17 for reporting this false positive. I've adjusted the rule, this will be fixed in the next release.

**Comment by peter17** [comment]
Awesome! Thanks!
